packages/expo-image-picker/src/ImagePicker.types.ts

```diff
@@ -220,6 +220,15 @@ export type ImageInfo = {
    * an image) and `width` and `height` specify the dimensions of the media.
    */
   uri: string;
+  /**
+   * The unique ID that represents the picked image or video. It can be used by [expo-media-library](./media-library)
+   * to manage the picked asset.
+   *
+   * > This might be `null` when the ID is unavailable or user gave limited permission to access the media library.
```

Comment:
```suggestion
   * The unique ID that represents the picked image or video. It can be used by [expo-media-library](./media-library)
   * to manage the picked asset.
   *
   * > This might be `null` when the ID is unavailable or the user gave limited permission to access the media library.
```